alexthompson207/current-climate
# Refactor/clean up code

# PR CurrentClimate

### What does this do?


### Are there any known bugs?


### Summary of the changes
Clean up code, remove code comments and console.logs. Re-run tests to make sure they still pass. Check props for each component and make sure they are being used. Change fav icon in browser tab.

### What ticket(s) do the changes resolve?
Closes #33 
Please Link Issues to this PR as necessary.

### Notes for the Reviewer
N/a


### How to Test Changes?
Make sure all tests still pass, `npm run cypress`
Icon in browser tab should now display a icon of earth rather than the react icon.


### Next Iterations
-README
-Deploy
